pgmpy/tests/test_factors/test_Factor.py

```diff
@@ -264,7 +264,8 @@ def test_eq1(self):
                                                       4, 5, 15, 16, 17, 6, 7,
                                                       8, 18, 19, 20, 9, 10, 11,
                                                       21, 22, 23])
-        self.assertTrue(phi1, phi2)
+        self.assertTrue(phi1 == phi2)
+        self.assertEqual(phi2.variables,['x2', 'x1', 'x3'] )
```

Comment:
@ankurankan This test is enough because, earlier it would have returned `['x1', 'x2', 'x3']`